book/src/config_api.md

```diff
@@ -161,6 +161,8 @@ generate_doc = false
         name = "website_label"
         # allow to remove/add Option<>
         nullable = true
+        # Take the parameter by value instead of by ref, only applies to the instance parameter
+        move = true
```

Comment:
Why? This should also work for other parameters, that would be mostly useless otherwise :) In GStreamer I need this for other parameters in almost all cases.

Reply:
Ah ok, I will change that then

Reply:
Should work for all the params now, do you mind trying it on some places in gstreamer to see if it behaves correctly?